<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6ST94J7" failures="1" tests="4" name="group-demo-1" time="0.094" errors="0" timestamp="2021-06-03T14:36:41 EDT">
  <testcase classname="com.testNGListenerDemo.GroupDemo1" name="test1" time="0.014"/>
  <testcase classname="com.testNGListenerDemo.GroupDemo1" name="test2" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [3] but found [5]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [5]
at com.testNGListenerDemo.GroupDemo1.test2(GroupDemo1.java:17)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="com.testNGListenerDemo.GroupDemo1" name="test3" time="0.0">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="com.testNGListenerDemo.GroupDemo1" name="test4" time="0.002"/>
</testsuite> <!-- group-demo-1 -->
